INSTALL.md

```diff
@@ -1,44 +1,56 @@
 
 **Windows**
 
-2019/2017: Download using the [GitHub .zip download](https://github.com/dracula/visual-studio/archive/master.zip) option and unzip them.
+- Latest/2022: Download the theme [from the Visual Studio Extensions Store (.vsix)](https://marketplace.visualstudio.com/items?itemName=dracula-theme.dracula).
+- Latest/2022: Download the theme [from GitHub (.zip)](https://github.com/dracula/visual-studio/archive/2022.zip) and unzip them.
 
-2015: Download using the [GitHub .zip download](https://github.com/dracula/visual-studio/archive/2015.zip) option and unzip them.
-
-2012: Download using the [GitHub .zip download](https://github.com/dracula/visual-studio/archive/2012.zip) option and unzip them.
+- 2019/2017: Download the theme [from GitHub (.zip)](https://github.com/dracula/visual-studio/archive/2019.zip) and unzip them.
+- 2015: Download the theme [from GitHub (.zip)](https://github.com/dracula/visual-studio/archive/2015.zip) and unzip them.
+- 2012: Download the theme [from GitHub (.zip)](https://github.com/dracula/visual-studio/archive/2012.zip) and unzip them.
 
 **Mac**
 
-Download using the [GitHub .zip download](https://github.com/dracula/visual-studio/archive/master.zip) option and unzip them.
+- Download the package [from GitHub (.zip)](https://github.com/dracula/visual-studio/archive/2019.zip) and unzip them.
 
-#### Activating theme
+### How to install and activate the theme
 
 **Windows**
 
-1.  Start Visual Studio
-2.  Go to `Tools -> Import and Export Settings... -> Import selected environment settings`
-3.  \[Choose whether you want to backup existing settings\]
-4.  Click `Browse` and select the .vssettings file
-5.  Click `Finish`
+1. Run the `DraculaTheme.vsix` file;
+2. Select the Visual Studio version you want to install the theme and click `Install`;
+3. With the installation complete, open your Visual Studio;
+4. Navigate to `Tools` -> `Theme` and tick `Dracula Theme`;
+5. Ready! 🚀
 
 **Mac**
 
-1.  Start Visual Studio
-2.  Go to `Preferences -> Text Editor -> Color Theme`
-3.  Click `Add` and then find the unzipped folder and select the .vssettings file
-4.  Click `Open`
-5.  Select the `dracula` theme option to preview it
-6.  Click `Okay` to apply it
+1. Start Visual Studio;
+2. Go to `Preferences` -> `Text Editor` -> `Color Theme`;
+3. Click "Add", find the unzipped folder and select the .vssettings file;
+4. Click "Open";
+5. Select the `Dracula Theme` theme option to preview it;
+6. Click "Ok" to apply it;
+7. Ready! 🚀
+
+> If you followed the steps above and could not install it, please [file an issue on GitHub](https://github.com/dracula/visual-studio/issues/new) or browse [our Discord](https://draculatheme.com/discord-invite), where the community can help.
+
+### Visual Studio
+
+If you have questions about Visual Studio, the best place to look for help is on the [official Visual Studio product family documentation](https://docs.microsoft.com/en-gb/visualstudio/).
```

Comment:
This whole section isn't needed and also doesn't follow the standard of other repos.